### PR TITLE
#242 Fix mention user selector

### PR DESCRIPTION
**Pull request by stephanevieira75**

### CODE REVIEW

File: twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx [hunk 1]
```diff
@@ -1,141 +1,175 @@
...
       resultPosition: '',
+      selectedUser: {},
+      focused: '',
     };
 
     this.keyUp = this.keyUp.bind(this);
     this.keyDown = this.keyDown.bind(this);
 
-    var that = this;
-    window.addEventListener('keydown', function (evt) {
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx:42]: Need to be changed to also remove the event listener on unmount

File: twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx [hunk 2]
```diff
@@ -1,141 +1,175 @@
...
+    window.addEventListener('keydown', (evt: any) => {
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx:67]: All addEventListener must be with a removeEventListener

Comment (RomaricMourgues) [reply twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx:67]: Here the keydown listener is used two time, no need to add two time this listener. move the content of this anonymous function into the "keyDown" method below

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]

File: twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx (hunk 1 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx:42]: Done :+1:

File: twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx (hunk 2 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/components/AutoComplete/AutoComplete.tsx]: Done :+1:

**Review by RomaricMourgues — APPROVED** [review]

**Pull request merged**